usecase/parser/ibcmsg/ibcmsg.go

```diff
@@ -881,6 +883,11 @@ func ParseMsgAcknowledgement(
 		ConnectionID:    acknowledgePacketEvent.MustGetAttributeByKey("packet_connection"),
 	}
 
+	// https://github.com/cosmos/ibc-go/blob/e98838612a4fa5d240e392aad3409db5ec428f50/modules/apps/transfer/module.go#L364
+	if fungibleTokenPacketEvent.GetAttributeByKey("error") != nil {
+		params.MaybeFungibleTokenPacketData.MaybeError = fungibleTokenPacketEvent.MustGetAttributeByKey("error")
```

Comment:
If there's no error, if this error string be present and empty? If yes, following the above design of `Maybe*`, here it should check for those case and assign `nil` when it is empty.

Reply:
Yes, good catch! Will update it!